extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -83,7 +87,7 @@ class SchemaCompareTester {
 		};
 
 		let schemaCompareResult = await schemaCompareService.schemaCompare(operationId, source, target, azdata.TaskExecutionMode.execute, null);
-		this.assertSchemaCompareResult(schemaCompareResult, operationId);
+		this.assertSchemaCompareResult(schemaCompareResult, operationId, 4);
```

Comment:
I really wish Typescript supported named parameters...  Don't like how we're forced into magic constants.

Reply:
For this I would suggest at least making a const with that value so that it's clear (plus that way updating one will update all the places it's used so less manual effort)

---
In reply to: [361736737](https://github.com/microsoft/azuredatastudio/pull/8765#discussion_r361736737) [](ancestors = 361736737)